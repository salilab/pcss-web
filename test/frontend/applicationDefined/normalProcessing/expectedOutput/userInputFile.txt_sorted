A5A6J1 157 LSVDYGKK 
A5PK01 303 LPGDNVGF 
BARKAN 46  EEEDEEEE
O00273 114 DETDSGAG 
O00273 3 VTGDAGVP 
O14641 18 YHLDEEET 
O95155 120 VDVDSGIE 
P04004 63 TRGDVFTM 
P06748 158 LAADEDDD 
P11387 12 IEADFRLN 
P12081 45 LGPDESKQ 
P12956 76 ISSDRDLL 
P14314 357 PPYDEQTQ 
P19338 238 EEEDDEDE
P19338 420 EEEDEEEE 
P50502 89 IEPDTDAP 
P55957 72 IEADSESQ 
P56192 675 TPDDQRLL 
P78316 366 GGEDTEES 
P78316 850 LKRKKFKK
P78316 851 LKRKKFKK
Q05CV4 178 RSRDRNRD 
Q53EP6 168 AVGDGDGD 
Q53EP6 170 GDGDGDGD 
Q53EP6 189 WDADAAEA 
Q76N46 206 RGVDYNAE 
