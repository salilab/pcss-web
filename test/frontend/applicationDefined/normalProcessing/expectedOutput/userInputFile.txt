O00273 3 VTGDAGVP 
P11387 12 IEADFRLN 
P12081 45 LGPDESKQ 
P04004 63 TRGDVFTM 
P55957 72 IEADSESQ 
P12956 76 ISSDRDLL 
P50502 89 IEPDTDAP 
O00273 114 DETDSGAG 
O95155 120 VDVDSGIE 
P06748 158 LAADEDDD 
A5A6J1 157 LSVDYGKK 
P14314 357 PPYDEQTQ 
O14641 18 YHLDEEET 
P56192 675 TPDDQRLL 
A5PK01 303 LPGDNVGF 
P19338 238 EEEDDEDE
P19338 420 EEEDEEEE 
Q53EP6 189 WDADAAEA 
Q76N46 206 RGVDYNAE 
Q05CV4 178 RSRDRNRD 
P78316 366 GGEDTEES 
P78316 851 LKRKKFKK
P78316 850 LKRKKFKK
Q53EP6 168 AVGDGDGD 
Q53EP6 170 GDGDGDGD 
BARKAN 46  EEEDEEEE
